data/json/recipes/armor/torso.json

```diff
@@ -170,6 +170,25 @@
       [ [ "faux_fur", 12 ] ]
     ]
   },
+  {
+    "result": "coat_gut",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_TORSO",
+    "skill_used": "tailor",
+    "skills_required": [ "survival", 5 ],
+    "difficulty": 5,
+    "time": 200000,
+    "autolearn": true,
+    "tools": [
+      [ [ "sewing_kit", -1 ] ]
```

Comment:
We have multiple items with SEW quality, so you better use quality instead of certain item.

Reply:
Done! Added Cutting as a req too for good measure.